pkgs/liquid-swap/default.nix

```diff
@@ -0,0 +1,17 @@
+{ lib, buildPythonPackage, fetchurl, pyqt5, click }:
+
+buildPythonPackage rec {
+  pname = "liquid-swap";
+  version = "0.0.1";
+
+  # We're only interested in the cli and not the gui but we need to add pyqt5
+  # anyway as a build dependency because liquid-swap's setup.py demands it. See
+  # issue https://github.com/Blockstream/liquid-swap/issues/1.
+  checkInputs = [ pyqt5 ];
```

Comment:
nativeBuildInputs seems more appropriate here

From the nixpkgs manual:
>  nativeBuildInputs ? []: Build-time only dependencies. Typically executables as well as the items listed in setup_requires. 
> checkInputs ? []: Dependencies needed for running the checkPhase. These are added to nativeBuildInputs when doCheck = true. Items listed in tests_require go here.

Reply:
fixed